api/models/fixtures/cron_labels.yml

```diff
@@ -7,4 +7,4 @@ main:
 really-long-process-type-name:
   build: .
   labels:
-    - convox.cron.really-long-cron-job-name=0 * * * ? bin/myjob
```

Comment:
Thinking now we should allow anything in a cron job name that is a valid Docker label key:

> * Keys should only consist of lower-cased alphanumeric characters, dots and dashes (for example, [a-z0-9-.]).
> * Keys should start and end with an alpha numeric character.
> * Keys may not contain consecutive dots or dashes.

https://docs.docker.com/engine/userguide/labels-custom-metadata/